### ISSUE TITLE
[Contributing] Missing information in contribution guidelines

### ISSUE THREAD

**Issue by Maniekko** [MonoGame/docs.monogame.github.io#60]
[There's a unfinished part](https://github.com/MonoGame/docs.monogame.github.io/blame/e61f2171a0af97db9cf436a595c4c4f7db884bba/articles/contributing.md#L44) in documentation contributing guidelines on for how to write manuals, guides, and tutorials.

**Comment by SimonDarksideJ** [comment]
Good point, need to review with the team on these updated guidelines now the main branch of work is done.

**Comment by SimonDarksideJ** [comment]
Resolved
